scala/lambdas/ingest-mapper/src/main/scala/uk/gov/nationalarchives/ingestmapper/DiscoveryService.scala

```diff
@@ -79,11 +83,11 @@ object DiscoveryService {
           body <- Async[F].fromEither(response.body)
           potentialAsset = body.assets.find(_.citableReference == citableReference)
           formattedAsset <- potentialAsset.map(stripHtmlFromDiscoveryResponse).getOrElse {
-            Async[F].pure(DiscoveryCollectionAsset(citableReference, DiscoveryScopeContent(""), None))
+            Async[F].pure(DiscoveryCollectionAsset(citableReference, DiscoveryScopeContent(None), None))
           }
         } yield formattedAsset
       }.handleErrorWith { e =>
-        logger.warn(e)("Error from Discovery") >> Async[F].pure(DiscoveryCollectionAsset(citableReference, DiscoveryScopeContent(""), None))
+        logger.warn(e)("Error from Discovery") >> Async[F].pure(DiscoveryCollectionAsset(citableReference, DiscoveryScopeContent(None), None))
```

Comment:
Should/Could `DiscoveryCollectionAsset(citableReference, DiscoveryScopeContent(None), None)` be saved to a lazy val outside of the for comprehension and then referred to on line 86 and 90?

Reply:
Yeah, see the other comment, I've made a method to create it.